src/main/kotlin/no/nav/familie/ef/sak/ekstern/bisys/BisysBarnetilsynService.kt

```diff
@@ -85,4 +86,24 @@ class BisysBarnetilsynService(
                 }
         return barnetilsynBisysPerioder
     }
+
+    private fun slåSammenPerioder(
+        infotrygdPerioder: List<BarnetilsynBisysPeriode>,
+        efPerioder: List<BarnetilsynBisysPeriode>
+    ): List<BarnetilsynBisysPeriode> {
+        if (efPerioder.isEmpty()) {
+            return infotrygdPerioder
+        }
+        val startdato = efPerioder.first().periode.fom
```

Comment:
startdato burde brukes fra tilkjent ytelse for å bli riktig her.
Hvis man eks har fjernet alle perioder fra EF. så blir alle perioder med fra infotrygd, som kan være helt feil

Reply:
Er startdato det samme som evt første fom dato ? Blir ikke periodene returnert også i InternPeriodeUtil hvis startdato ikke er angitt ?